cli/cli.py

```diff
@@ -0,0 +1,155 @@
+"""
+CLI for interacting with the REMOTE analysis-runner
```

Comment:
Can we move this to a README.md instead? I find that a bit more discoverable than in the source code.

Reply:
Yep, I've created a README